el/sclang-help.el

```diff
@@ -24,15 +24,29 @@
 (require 'sclang-interp)
 (require 'sclang-language)
 (require 'sclang-mode)
-(require 'sclang-vars)
+(require 'sclang-vars nil 'ignore-missing-file)
 (require 'sclang-minor-mode)
 
-(defcustom sclang-help-directory "~/SuperCollider/Help"
-  "*Directory where the SuperCollider help files are kept. OBSOLETE."
+(defun sclang-system-root ()
+  "Find the common install location for the platform."
+  (cond
+   ((boundp 'sclang-system-data-dir)
+    sclang-system-data-dir)
+
+   ((eql 'darwin system-type)
+    (expand-file-name "~/Library/Application Support/SuperCollider"))
+
+   ((and (eql 'gnu/linux system-type)
+         (file-exists-p "/usr/local/share/SuperCollider"))
+    "/usr/local/share/SuperCollider")
+
+   ((eql 'gnu/linux system-type)
+    "/usr/share/SuperCollider")))
```

Comment:
I'd prefer to leave this out of this PR because
- I don't know of anyone who could test this
- It can easily be set via `customize`

Hopefully some day a windows user will send us a PR :)